src/Execution/Arguments/NestedManyToMany.php

```diff
@@ -59,4 +59,33 @@ public function __invoke($parent, $args)
             );
         }
     }
+
+    /**
+     * @param  \Nuwave\Lighthouse\Execution\Arguments\Argument $args
+     *
+     * @return array
+     */
+    private function generateRelationArray(Argument $args)
+    {
+        $values = $args->toPlain();
+
+        if (empty($values)) {
+            return [];
+        }
+
+        if (! is_array($values[0])) {
```

Comment:
This check might be more straightforward by examining the `Argument` class, it is meant for use cases like these where we need to check the type of arguments during query execution.

Reply:
Ok, can you say, what I have to compare? `$args->type` can't help there much, as it is in both cases a `ListType`, and the type itself is just a string, like `ID`, but we can't take comparison to `ID`, as I e.g. using `ObfID` in my project as main ID type. 

We could also check the value of `$args->value[0]` if it is an object or not. What do you think?

Reply:
I tried out the other approach and found your initial solution to be simpler. Well done.